portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/SkinnableCharonPortal.java

```diff
@@ -578,6 +519,7 @@ public PortalRenderContext includePortal(HttpServletRequest req,
 			siteView.setIncludeSummary(includeSummary);
 			siteView.setDoPages(doPages);
 			if ( !siteView.isEmpty() ) {
+				System.out.println(siteView.getRenderContextObject());
```

Comment:
the log is intentional?

Reply:
No. Well spotted. System.out should never be left in the code :)

Reply:
Fixed